### ISSUE TITLE
privacy policy?

### ISSUE THREAD

**Issue by martey** [david-sabata/web-scrobbler#264]
This extension [seems to use Google Analytics for tracking](https://github.com/david-sabata/Chrome-Last.fm-Scrobbler/blob/54699b419af6fa48e34af911f4e894808740b451/background-ga.js), but I could not find any documentation that mentions what data is collected, how it is used, and/or who it is accessible to.

Both the [Google Chrome Web Store Terms of Service](https://developers.google.com/chrome/web-store/terms) and the [Google Analytics Terms of Service](http://www.google.com/analytics/terms/us.html) seem to mandate use of a privacy policy.

**Comment by martey** [comment]
> There really isn't much to say, because the Analytics are used at its minimum

I was auditing my Chrome extensions today and saw that more data is being sent to Google Analytics than before. It looks like d353cc9d9fc69877b0adb15e879b566672fdf5cc and 0e207145d5a7d1dec74c1eece0dd5caa07917eff increased the amount of Google Analytics tracking, and that behavior around notifications and **the artist and track being scrobbled is now being tracked**.

Despite the fact that these changes were made in June and September of 2014, the privacy policy has not been updated to account for them. Even if it had, it's doubtful that anyone installing it would notice, since the privacy policy is still not linked from anywhere in the extension's description in the Chrome Web Store.

I really think:

 - the privacy policy should be linked from the extension's description in the Web Store and on the project's README
 - users should be prompted upon installation whether they want to enable Google Analytics tracking
 - **all existing data in Google Analytics should be deleted**, since none of the users who submitted it agreed to have it collected

**Comment by martey** [comment]
Hi @david-sabata, any comment on the pull requests that @inverse made?

**Comment by david-sabata** [comment]
Hey @martey. To be honest I don't know why I added tracking of artist and track. It's like a year ago and  I don't think I looked at the data once. So I removed the code - 2e6646e12

As for the other commit you reference - it tracks notifications showing up, being clicked or failing. I believe that falls into "the extension usage tracking" noted in the privacy policy.

I also merged opt-out checkbox in settings and link to the privacy policy on the same page. 

And last thing - GA data. I haven't found a way to selectively remove only the events data and I definitely don't agree that all data in GA should be removed.
